spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:example) { get('/users/745/posts') }
+    it 'check status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include 'Posts of user'
+    end
+  end
+
+  describe 'GET /index' do
+    before(:example) { get('/users/745/posts/4') }
+    it 'check status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template('show')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include 'Posts 4 by user'
+    end
+  end
+end
```

Comment:
- [ ] Check that it does not render a different template
- [ ] Modify `expect(response.body).to include 'Posts 4 by user'` with `expect(response.body).to include ("Posts 4 by user")`

Reply:
Hi @Emmyn5600, thanks for mentioning the issues. I have fixed these.

Reply:
These issues have been addressed well. Good Job! :+1: :+1: